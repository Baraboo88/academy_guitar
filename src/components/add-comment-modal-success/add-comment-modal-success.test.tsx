import * as renderer from 'react-test-renderer';
import * as React from 'react';


import AddCommentModalSuccess from './add-comment-modal-success';

it('AddCommentModalSuccess successfully rendered', () => {
  const mockCloseHandler = jest.fn();
  const tree = renderer.create(<AddCommentModalSuccess onCloseModalHandler={mockCloseHandler}/>);
  expect(tree).toMatchSnapshot();
});
