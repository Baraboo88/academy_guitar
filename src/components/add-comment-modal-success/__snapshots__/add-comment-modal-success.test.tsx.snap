// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCommentModalSuccess successfully rendered 1`] = `
<div
  className="modal is-active modal--success modal-for-ui-kit"
>
  <div
    className="modal__wrapper"
  >
    <div
      className="modal__overlay"
      data-close-modal={true}
      data-test="test-close-modal"
      onClick={[MockFunction]}
    />
    <div
      className="modal__content"
    >
      <svg
        aria-hidden="true"
        className="modal__icon"
        height="20"
        width="26"
      >
        <use
          xlinkHref="#icon-success"
        />
      </svg>
      <p
        className="modal__message"
      >
        Спасибо за ваш отзыв!
      </p>
      <div
        className="modal__button-container modal__button-container--review"
      >
        <button
          className="button button--small modal__button modal__button--review"
          data-test="test-close-modal"
          onClick={[MockFunction]}
        >
          К покупкам!
        </button>
      </div>
      <button
        aria-label="Закрыть"
        className="modal__close-btn button-cross"
        data-test="test-close-modal"
        onClick={[MockFunction]}
        type="button"
      >
        <span
          className="button-cross__icon"
        />
        <span
          className="modal__close-btn-interactive-area"
        />
      </button>
    </div>
  </div>
</div>
`;
